### ISSUE TITLE
[A11y]

### ISSUE THREAD

**Issue by msxlove** [IBMa/equal-access#232]
## Environment

> Operating system:

> Browser:

> Automated testing tool and ruleset:

> Assistive technology used to verify:

## Detailed description

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

- Screenshots or code
- Notes

**Comment by tombrunet** [comment]
Closing since there's no information provided in this issue
